### PR TITLE
logging in case of success in HTTPAgent

### PR DESCRIPTION
**Pull request by aleesha711**

### CODE REVIEW

**Comment by aleesha711** [comment]
@githengi @keymane

File: opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java [hunk 1]
```diff
@@ -147,9 +147,15 @@ private HttpURLConnection initializeHttp(String requestURLPath, boolean useBasic
...
     public Response<String> postWithJsonResponse(String postURLPath, String jsonPayload) {
+        logResponse(postURLPath, jsonPayload);
         return post(postURLPath, jsonPayload);
     }
 
+    private void logResponse(String postURLPath, String jsonPayload) {
+        Timber.d("postURLPath: %s", postURLPath);
+        Timber.d("jsonPayLoad: %s", jsonPayload);
+    }
+
```
Comment (githengi) [inline opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java]: Can these be combined to one once both the url and the payload

Comment (aleesha711) [reply opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java]: @githengi its done! please check

Comment (aleesha711) [reply opensrp-app/src/main/java/org/smartregister/service/HTTPAgent.java]: @githengi can you review please? Thanks in advance

**Comment by rahimkhalid** [comment]
@githengi @keymane

**Comment by aleesha711** [comment]
@githengi can you check please

**Pull request merged**